src/ai/backend/test/testcases/session/destroy_session.py

```diff
@@ -0,0 +1,52 @@
+import asyncio
+import json
+
+from ai.backend.client.session import AsyncSession
+from ai.backend.test.templates.template import TestCode
+from ai.backend.test.testcases.session.context import ComputeSessionContext
+
+# Test environment configuration
+# TODO: Make these configurable loaderable by template wrapper
+_TEST_TIMEOUT = 30.0  # seconds
```

Comment:
I'm considering whether to receive timeout as a contextvar as well. (A method of using a default timeout if the contextvar is not set?)